src/generateTrackerData.ts

```diff
@@ -0,0 +1,60 @@
+import { IAnalyticsTrackerOptions } from './types';
+
+export const generateTrackerData = (
```

Comment:
How do you wanna use `generateTrackerData`? Would you mind adding some tests to show how to use it, please.

Reply:
My idea is leaving the [big switch statement](https://github.com/Zizzamia/perfume.js#quick-start) reported in the Readme inside the Perfume.js repo. This way every future change to the tracked metrics requires just an update of that function, without affecting my plugin that should include the big switch statement into its code and it needs to be kept in sync at every change.

I'd consume it importing it in my plugin and "printing" it (with a `toString()` call) on the page [here](https://github.com/NoriSte/gatsby-plugin-perfume.js/blob/master/src/gatsby-ssr.js#L28).

If the idea could work for you I will add all the necessary tests 😉